pep257.py

```diff
@@ -219,6 +232,7 @@ def __call__(self, filelike, filename):
         self.stream = TokenStream(StringIO(src))
         self.filename = filename
         self.all = None
+        self._decorators = []
```

Comment:
Maybe this should be named `_current_decorators` or something of this sort. At first I was confused as to why this is inside `Parser`.